kdc/kerberos5.c

```diff
@@ -1347,6 +1347,10 @@ get_pa_etype_info2(krb5_context context,
     if(pa.val == NULL)
 	return ENOMEM;
 
+    if (config->always_include_salt) {
+	include_salt = TRUE;
+    }
```

Comment:
no braces necessary

Reply:
Sorry about that, Samba coding conventions creeping in.